config/routes.rb

```diff
@@ -1,8 +1,18 @@
+class AuthConstraint
+  def matches?(request)
+    request.session[:user]["email"].present?
+  end
+end
+
 Rails.application.routes.draw do
-  root to: "scheduled_messages#index"
+  constraints(AuthConstraint.new) do
+    root to: "scheduled_messages#index"
```

Comment:
nice! can we write a little spec for this? maybe something similar to https://github.com/18F/dolores-landingham-bot/blob/master/spec/requests/auth_spec.rb but that makes sure a logged in user is taken to this page and a non-logged in user is not?